lib/utils.ts

```diff
@@ -425,7 +425,7 @@ export function countOccurrences<T>(collection: Iterable<T>, item: T): number {
 }
 
 export function asSafeVer(semver: string | number | null) {
-    if (semver !== null) {
+    if (semver != null) {
```

Comment:
What's the distinction here? (My spider sense goes off without two `==`s) Could this be `if (semver) {` and still work if we just want "truthiness" ?

Reply:
This way this also catches `undefined` which I was somehow getting. Should probably update the types too.
(We don't want truthiness because we want to get inside in case of number 0, for example)

Reply:
So, basically, https://eslint.org/docs/latest/rules/eqeqeq#smart